app/lib/services/handle_submission_for_store.rb

```diff
@@ -198,5 +198,13 @@ def school
     def user
       @user ||= User.find_by(email: store["confirmed_email"])
     end
+
+    def target_cohort
+      if course.ehco?
+        2021
+      else
+        Application::DEFAULT_COHORT
+      end
```

Comment:
This feels like it should be something being returned by the course, rather than being calculated inside service.

Reply:
it's now a field/column on the course